proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -354,3 +357,12 @@ message PayloadSCIMTokenCreated {
 }
 
 message PayloadSCIMTokenDeleted {}
+
+message PayloadDeleteRepositoryDraftCommit {
```

Comment:
This should be exactly the same as `PayloadRepositoryCommitPushed`, but `PayloadRepositoryCommitDeleted`